packages/extension-ui/src/utils/buildHierarchy.ts

```diff
@@ -0,0 +1,23 @@
+import { AccountJson, AccountWithChildren } from '@polkadot/extension-base/background/types';
```

Comment:
Header missing.

Reply:
Maybe consider adding this plugin to linter?
https://www.npmjs.com/package/eslint-plugin-header

Reply:
Will add it. I hate playing manual linter. (And I certainly also miss a lot of stuff when I code and then have to shout at myself down the line)